config_utilities.hpp

```diff
@@ -472,8 +475,13 @@ struct ConfigInternal : public ConfigInternalVerificator {
     return result;
   }
 
-      [[nodiscard]] std::string toString() const {
+  std::string toString() const {
     meta_data_->messages = std::make_unique<std::vector<std::string>>();
+    meta_data_->merged_setup_already_used = true;
+    meta_data_->merged_setup_set_params = false;
+    meta_data_->merged_setup_currently_active = true;
+    // NOTE: setupParamsAndPrinting() does not modify 'this' in printing mode.
+    ((ConfigInternal*)this)->setupParamsAndPrinting();
```

Comment:
Unfortunately we're casting away constness here which is not allowed for C++ casts afaik. It's ugly but since it's only internal nothing bad can happen and it makes the tool a lot more userfriendly (: